reloj-checador/app/views/employees/_form.html.erb

```diff
@@ -1,4 +1,6 @@
 <%= form_with(model: employee) do |form| %>
+
+<!--
```

Comment:
I think this is not needed he he

Reply:
I did not realize that xd, thaks for the resources